samples/Compute/allowed-platform-images/azurepolicy.json

```diff
@@ -21,27 +29,15 @@
                             "allOf": [
                                 {
                                     "field": "Microsoft.Compute/imagePublisher",
-                                    "in": [
-                                        "Canonical"
-                                    ]
+                                    "in": ["parameters:('imagePublishers')"]
```

Comment:
should be:
`"in": "[parameters('allowedImagePublishers')]"`
etc

If you have trouble remembering the policy syntax for parameters, just check out the other samples!